test/images/volume/nfs/run_nfs.sh

```diff
@@ -22,21 +22,23 @@ function start()
     while getopts "G:" opt; do
         case ${opt} in
             G) gid=${OPTARG};;
+            *) echo "usage: $0 [-v] [-r]" >&2
```

Comment:
isn't the only accepted option -G?

Reply:
change to: `G) echo "usage: $0 [-v] [-r]" >&2` ?

Reply:
why -v -r? Those aren't the possible usage options.

Reply:
I am not very familiar with this document, may I ask if you have any good Suggestions for modification?

Reply:
I think `[-G]` is all you need, as that's the only possible option.

Reply:
Thank you both. Is that right?